geniza/corpus/templates/corpus/document_scholarship.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'base.html' %}
+{% load static i18n %}
+
+{% block title %}{{ document.title }}{% endblock title %}
+
+{% block main %}
+<!-- document scholarship records -->
+<div class="container">
+    <h1>{{ document.title }}</h1>
+    {% include "corpus/snippets/document_tabs.html" %}
+    <ol>
+        {% for fn in document.footnotes.all %}
+        {% spaceless %}
+        <li><p class="footnote">
+        {% if fn.source.title %}<span class="title">{{ fn.source.title }}</span>{% endif %}
+        {% if fn.source.authors.exists %}<span class="author">{{ fn.source.all_authors }}</span>{% endif %}
+        {% if fn.source.year %}<span class="year">{{ fn.source.year }}</span>{% endif %}
+        <span class="relation">{{ fn.doc_relation }}</span>
+        {% if fn.location %}<span class="location">{{ fn.location }}</span>{% endif %}
+        {% if fn.url %}<a href="{{ fn.url }}">{% translate "Link to source" %}</a>{% endif %}
+        </p></li>
+        {% endspaceless %}
+        {% endfor %}
```

Comment:
I see this implementation matches Gissoo's designs — we'll probably want to revise that now that we know more about what the source data looks like, but seems fine for current purposes.

Reply:
agree; it looks awkward for some current data but seemed OK for a first pass